exampleCourse/questions/demo/autograder/python/dragDropCode_simple/server.py

```diff
@@ -0,0 +1,5 @@
+def generate(data):
+    data['params']['names_for_user'] = []
+    data['params']['names_from_user'] = [
+        {'name': 'return_even', 'description': 'Function to compute the $n^\\text{th}$ Fibonacci number', 'type': 'python function'}
```

Comment:
This description seems wrong.

Reply:
Yes. I copied this from an example question and forgot to change the description

Reply:
now fixed